src/js/core/Core.js

```diff
@@ -58,6 +62,38 @@ const getPopupPromise = (requestWindow: boolean = true): Deferred<void> => {
     return _popupPromise;
 };
 
+/**
+ * Start interaction timeout timer
+ */
+const interactionTimeout = () => _timeout.start(async () => {
+    /**
+     * Bridge version =< 2.0.28 has a bug that doesn't permit it to cancel
+     * user interactions in progress, so we have to do it manually.
+     */
+    const { version } = _core.getTransportInfo();
```

Comment:
check only bridge transport (webusb not affected)
`const { type, version }`
`if (type === 'bridge' && versionCompare....`